app/src/main/res/layout/fragment_movie_list.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.MovieListFragment">
+
+<androidx.recyclerview.widget.RecyclerView
```

Comment:
You can have the Recycler view directly, without the frame layout. If you have only one view in a layout it is not necessary to put it in a container